vm/src/builtins/complex.rs

```diff
@@ -337,6 +337,11 @@ impl PyComplex {
         vm.ctx
             .new_tuple(vec![vm.ctx.new_float(re), vm.ctx.new_float(im)])
     }
+
+    #[pymethod(magic)]
+    fn str(&self, vm: &VirtualMachine) -> PyStrRef {
+        PyStr::from(self.repr()).into_ref(vm)
```

Comment:
`self.repr()` will be enough with return type `String`

Reply:
I think adding `#[pymethod(name = __str__)]` on `fn repr` will work

Reply:
Thank you. I sent v2 code 🥳